extensions/sql-migration/src/constants/strings.ts

```diff
@@ -82,8 +82,50 @@ export const XEVENTS_ASSESSMENT_DESCRIPTION = localize('sql.migration.database.a
+export const READINESS_SECTION_TITLE = localize('sql.migration.readiness.section.title', "Migration readiness of assessed databases in the Server instance");
+export function TOTAL_FINDINGS_LABEL(findingsCount: number) { return localize('sql.migration.total.findings.label', "Total findings:{0}", findingsCount) }
+export const ISSUES_LABEL = localize('sql.migration.issues.label', "Blocking issues");
+export const INSTANCE_FINDING_SUMMARY = localize('sql.migration.instance.finding.summary', "Server instance assessment findings summary");
+export const SEVERITY_FINDINGS_LABEL = localize('sql.migration.severity.findings.label', "Findings by severity");
+export function ASSESSED_DBS_LABEL(databasesCount: number) { return localize('sql.migration.assessed.dbs.label', "Assessed databases:{0}", databasesCount) }
+export const NOT_READY = localize('sql.migration.not.ready', "Not ready");
+export const READY = localize('sql.migration.ready', "Ready");
+export const NEEDS_REVIEW = localize('sql.migration.needs.review', "Needs review");
+export const READY_WARN = localize('sql.migration.ready.warn', "Ready with warnings");
+export const BLOCKERS = localize('sql.migration.ready', "Blockers");
+export const DATABASE_ISSUES_SUMMARY = localize('sql.migration.database.issues.summary', "Database assessment issues summary");
+export function TOTAL_ISSUES_LABEL(issuesCount: number) { return localize('sql.migration.total.issues.label', "Total issues found:{0}", issuesCount) }
+export const SEVERITY_ISSUES_LABEL = localize('sql.migration.severity.issues.label', "Issues by severity");
+export function DB_READINESS_SECTION_TITLE(dbName: string) {
+	return localize('sql.migration.db.readiness.section.title', "Database {0} migration readiness", dbName);
+}
+export function NON_READINESS_DESCRIPTION(issueCount: number) {
+	return localize('sql.migration.non.readiness.description', "The database is not ready to migrate due to {0} blocking issue(s).", issueCount);
+}
+export function WARNING_READINESS_DESCRIPTION(warnings: number) {
+	return localize('sql.migration.non.readiness.description', "The database is ready to migrate with {0} warnings.", warnings);
+}
+export const READINESS_DESCRIPTION = localize('sql.migration.readiness.description', "The database is ready to migrate.");
+export const SELECT_TARGET_LABEL = localize('sql.migration.no.target.selected.label', 'Select target type');
+export const NO_TARGET_SELECTED_LABEL = localize('sql.migration.select.target.label', 'No Target selected. Select Target to view the Assessment Summary.');
+export const FINDINGS_LABEL = localize('sql.migration.findings.label', "Findings");
+export const ENCRYPTION_LABEL = localize('sql.migration.encryption.label', "Encryption");
+export const ENCRYPTION_DESCRIPTION = localize('sql.migration.encryption.description', "This database is encrypted using Transparent data encryption (TDE). You need to migrate certificates before you can migrate this database.");
+
```

Comment:
Since TDE is a name, this should be cases as: "Transparent Data Encryption (TDE)..."

Reply:
done